pkg/manager/manager_test.go

```diff
@@ -114,7 +115,7 @@ func setUp() (*Manager, map[string]*change.Change, map[store.ConfigName]store.Co
 		},
 		make(chan events.TopoEvent, 10))
 	if err != nil {
-		fmt.Println(err)
+		log.Fatal(err)
```

Comment:
is this fatal preventing the -1 exit to the os effectively blocking the test and not failing it ?

Reply:
changed